Makefile

```
@@ -43,4 +43,4 @@ migrate: build-pbmigrate
 .PHONY: generate-migration
 MIGRATION_NAME?=unnamed
 generate-migration:
-	migrate create -ext sql -dir api/migrations -seq -digits 3 $(MIGRATION_NAME)
\ No newline at end of file
+	migrate create -ext sql -dir api/migrations -seq -digits 3 $(MIGRATION_NAME)
```

```suggestion
	migrate create -ext sql -dir internal/db/migrations -seq -digits 3 $(MIGRATION_NAME)
```

@adiabramovitch spotted this, since you are touching this can you also fix it to prevent merge conflict :)

---

I've dropped the change as I've just added the endline.